app/models/post.rb

```diff
@@ -1,4 +1,8 @@
 class Post < ApplicationRecord
+  validates :title, presence: true,
+                    length: { maximum: 250, too_long: 'must not exceed 250 characters' }
+  validates :comments_counter, :likes_counter, numericality: { greater_than_or_equal_to: 0 }
```

Comment:
- [ ] CommentsCounter must be an integer greater than zero.

Reply:
- [ ] I think you are missing `Likes_counter` in your model and it must be an integer greater than zero.

Reply:
There's a likes_counter in my Model @DeliceLydia

Reply:
```js
validates :comments_counter, :likes_counter,
```